src/index.js

```diff
@@ -2,18 +2,25 @@ const isBrowser = typeof require === 'undefined'
 const fetch = isBrowser ? window.fetch : require('node-fetch')
 
 class PublicGoogleSheetsParser {
-  constructor (spreadsheetId) {
+  constructor (spreadsheetId, tabName) {
     this.id = spreadsheetId
+    this.tab = tabName
```

Comment:
I think it would be better to use sheetName rather than tab for this variable name.